bip-0182.md

```diff
@@ -0,0 +1,369 @@
+```
+  BIP: 182
+  Layer: Peer Services
+  Title: Utreexo - Transaction and block validation
+  Author: Tadge Dryja <rx@awsomnet.org>
+          Calvin Kim <bip@calvinkim.info>
+          Davidson Souza <bip@dlsouza.dev>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0182
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-10-01
+  License: BSD-3-Clause
+  Requires: 181
+```
+
+## Abstract
+
+This BIP defines the rules for validating blocks and transactions using the
+Utreexo accumulator. It is important to note that this BIP does not define the
+Utreexo accumulator itself, for that see [Utreexo Accumulator Specification](bip-0181.md). This document is only concerned with
+the general rules for validating blocks and transactions using the Utreexo,
+so that all Utreexo nodes can stay in consensus with one another.
+
+## Motivation
+
+Although Utreexo in its current form is not proposed as a soft fork, it is essential that
+all implementations adhere to a consistent workflow when performing consensus-critical
+operations. This BIP defines that workflow, along with the specific rules and their
+required ordering.
+
+There are five consensus-critical components when using the Utreexo accumulator to
+represent the UTXO set:
+
+ - 1: The serialization format of each UTXO ("leaf data").
+ - 2: The hash function used to hash the leaf data.
+ - 3: Which transaction outputs are excluded from the accumulator.
+ - 4: The order of operations for the additions and deletions in the accumulator.
+ - 5: The format of the UTXO proof.
+
+A discrepancy in any of the five components above will result in a divergent
+accumulator state, leading to consensus incompatibilities.
+
+## License
+
+This BIP is licensed under the BSD 3-clause license.
+
+## Specification
+
+### Node Hashes
+
+During a node's normal operation, it will need to compute the leaf hash for UTXOs
+being added or removed from the accumulator. The leaf hash is a 32-byte hash that
+is computed using the SHA-512/256 hash function. See [UTXO Hash Preimages](#utxo-hash-preimages) for the
+details on how to compute the leaf hash.
+
+Unless otherwise specified, all fields are in little-endian format.
+
+#### UTXO Hash Preimages
+
+Individual UTXOs are represented as 32-byte hashes in the Utreexo accumulator. To obtain this
+hash, you must compute the SHA-512/256 hash of the following data:
+
+| Name               | Type                     | Description                               |
+| ------------------ | ------------------------ | ----------------------------------------- |
+| Utreexo_Tag_V1     | 64-byte array            | The version tag to be prepended to the leafhash. |
+| Utreexo_Tag_V1     | 64-byte array            | The version tag to be prepended to the leafhash. |
```

Comment:
Duplicate table rows.

Reply:
Sorry, didn't notice this was intentional. You could consolidate this into one row that says "128-byte array", or change the description of the second line to clarify the intended duplication.